src/Core/Grid/Action/Row/Type/SubmitRowAction.php

```diff
@@ -72,10 +73,14 @@ public function configureOptions(OptionsResolver $resolver)
      */
     public function isApplicable(array $record)
     {
-        $options = $this->getOptions();
+        $accessibilityChecker = $this->getOptions()['accessibility_checker'];
 
-        if (is_callable($options['is_applicable_callback'])) {
-            return call_user_func($options['is_applicable_callback'], [$record]);
+        if ($accessibilityChecker instanceof AccessibilityCheckerInterface) {
```

Comment:
looks like we could have a hook here for instance, and the only expected return is a boolean so people can define their own rules in module.

just an idea: I don't think we should do it right now (we need feedbacks from developers after 1.7.5 release)